model-optimizer/extensions/middle/FusedBatchNormTraining.py

```diff
@@ -44,14 +48,33 @@ class FusedBatchNormTraining(MiddleReplacementPattern):
     def pattern(self):
         return dict(
             nodes=[
-                ('op', dict(kind='op', op=lambda op: op in ['FusedBatchNorm', 'FusedBatchNormV2', 'FusedBatchNormV3'],
-                 is_training=True))],
+                ('op', dict(kind='op', op=lambda op: op in ['batchNormTraining']))],
             edges=[]
         )
 
     def replace_pattern(self, graph: Graph, match: dict):
-        node = match['op']
-        node.is_training = False
+        bn_train_node = match['op']
+        additional_attrs = {}
+        for batchNormAttr in batchNormAttrList:
+            if bn_train_node.has(batchNormAttr):
+                additional_attrs[batchNormAttr] = bn_train_node[batchNormAttr]
+
+        if len(bn_train_node.out_nodes().items()) > 1:
+            additional_attrs['name'] = bn_train_node.name + '/batchNormInferenceMO'
```

Comment:
Do not use `bn_train_node.name`. Instead create a variable `bn_train_node_name = bn_train_node.soft_get('name', bn_train_node.id)` and use it everywhere in the code

Reply:
done